data/remnant/remnant 2 cognizance.txt

```diff
@@ -0,0 +1,1616 @@
+	on complete
+		log "Successfully introduced a flock of void sprites to Esquiline. Time will tell how they fare."
+		"salary: Remnant" = 2500
+		payment 1000000
+		event "remnant: pantica void sprites"
+		event "remnant: cognizance timer" 30
+		conversation
+			`	Moments later a ping comes in on the commlink, and Prefect Chilia and Plume appear onscreen. "Thank you for your work, Captain. It is unclear what the results of this effort will be, but for now, we wait and see. Hopefully the void sprites settle in," concludes Plume.`
+			`	"In the meantime," continues Chilia. "We are allocating you a daily stipend from our resources to cover some of your ongoing crew, maintenance, and upgrading costs. A larger one-time allotment has already been credited to your account with the shipyard facilities to help cover costs incurred during this mission."`
+
+
+
+mission "Remnant: Cognizance 19"
+	landing
+	name "Deliver navigation data"
+	description "Deliver navigational data on the new systems to <destination>."
+	source
+		government "Remnant"
+	destination
+		government "Remnant"
+		distance 1 5
+	to offer
+		has "Remnant: Cognizance 18: done"
+	on offer
+		conversation
+			`As you finish your landing checks, your commlink notifies you of an incoming message from Dawn:`
+			`	"Greetings, Captain <first>! We will be compiling updated navigational charts for the new systems and would like a fresh copy of the data your nav system recorded. We are hoping to retrieve some additional data that is not routinely downloaded by the automated starport systems. Stop by <planet>, and the starport systems will copy the data during the systems check. It's a pretty small data set, so there's no need to use data crystals. Thank you!"`
+			`	The end of the message is the location of a starport on <planet>, presumably where you are supposed to go.`
+				accept
+	on complete
+		payment 100000
+		event "remnant: cognizance 1 systems available"
+		dialog `As the starport computer downloads the data, your computer displays a message tacked onto the log file: "Thank you for providing the additional data from your navigational logs. It will be useful to contrast against those of our scouts. A logistics adjustment of <payment> has been authorized for the <ship>."`
+
+
+
+mission "Remnant: Cognizance 20"
+	name "Preparation for expedition"
+	description "Outfit a research laboratory, retrieve equipment from <stopovers>, then bring it to <destination>."
+	blocked "You need 10 free cargo space as well as a research laboratory for this mission."
+	cargo "supplies" 10
+	source
+		government "Remnant"
+	stopover
+		government "Remnant"
+		neighbor distance 1 5
+	to offer
+		has "Remnant: Cognizance 19: done"
+		has "event: remnant: lab availability"
```

Comment:
To the best of my knowledge and research of the files, the player can get here without having acquired the lab. As such, this is another spot where the mission chain can end abruptly without the player knowing why - and `blocked` dialog won't appear because the player hasn't even met the offer criteria. I'm cool with letting it slide for the sake of merging this PR, but let's make a note to come back to this. Maybe another mission that can offer when these criteria aren't met at least explaining why the player can't continue the story?